content/en/real_user_monitoring/connect_rum_and_traces/_index.md

```diff
@@ -182,10 +184,39 @@ The following Datadog tracing libraries are supported:
 | [.NET][14]                  | [1.18.2][15]                |
 
 
+## Open Telemetry support
+
+RUM also supports several propagator types to connect resources with backends instrumented with Open Telemetry libraries.
+
+{{< tabs >}} {{% tab "Browser RUM" %}}
+1. Setup RUM to connect with APM as described above
+
+2. Modify allowedTracingUrls such as:
+    ```javascript
+    import { datadogRum } from '@datadog/browser-rum'
+
+    datadogRum.init({
+        ...otherConfig,
+        allowedTracingUrls: [
+          { match: "https://api.example.com", propagatorTypes: ["tracecontext"]}
+        ]
+    })
+    ```
+    `match` accepts the same parameter types (string, RegExp or function) as when used in its simple form.
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏
```suggestion
    `match` accepts the same parameter types (`string`, `RegExp` or `function`) as when used in its simple form.
```